server.go

```diff
@@ -2821,8 +2822,14 @@ func (s *server) Peers() []*peer {
 // form "#RRGGBB", parses the hex color values, and returns a color.RGBA
 // struct of the same color.
 func parseHexColor(colorStr string) (color.RGBA, error) {
-	if len(colorStr) != 7 || colorStr[0] != '#' {
-		return color.RGBA{}, errors.New("Color must be in format #RRGGBB")
+	// Parses a regular expression and returns a Regexp object.
+	// Panics if the expression cannot be parsed.
+	validColorRegexp := regexp.MustCompile("^#[A-Fa-f0-9]{6}$")
```

Comment:
would recommend moving this to a global variable, instead of calling `MustCompile` in this method. rationale is that we'd rather `MustCompile` panic when initializing the runtime, as opposed to when the function is invoked